src/sgame/sg_main.cpp

```diff
@@ -213,6 +213,8 @@ Cvar::Cvar<float>  g_devolveMaxBaseDistance("g_devolveMaxBaseDistance", "Max Ove
 
 Cvar::Cvar<bool>   g_autoPause("g_autoPause", "pause empty server", Cvar::NONE, false);
 
+Cvar::Cvar<int> g_maxMiners("g_maxMiners", "set maximum number of miners per team. 0 = disabled.", Cvar::NONE, 0);
```

Comment:
Since using an int, maybe use `0` for exactly that: 0 miners allowed, and negative value in cvar to disable the feature?

Reply:
If there is any use for a game without miners, which I assume: you are right.